example/lib/pages/markers.dart

```diff
@@ -110,9 +110,11 @@ class _MarkerPageState extends State<MarkerPage> {
               options: MapOptions(
                 initialCenter: const LatLng(51.5, -0.09),
                 initialZoom: 5,
-                onTap: (_, p) => setState(() => customMarkers.add(buildPin(p))),
+                onTap: (_, p) {
+                  setState(() => customMarkers.add(buildPin(p)));
+                },
                 interactionOptions: const InteractionOptions(
-                  flags: ~InteractiveFlag.doubleTapZoom,
+                  enabledGestures: EnabledGestures.noRotation(),
```

Comment:
Why not use `EnabledGestures.all(doubleTapZoomIn: false)` like in the file `latlng_to_screen_point.dart`? Or maybe it should be the other way around and use `EnabledGestures.noRotation()` as well?

Reply:
Oh using the noRotation constructor is wrong in the first place. I think the reason to disable double tap gestures was to remove the click delay but I make a mistake while migrating.
Are you fine to remove the custom `interactionOptions` here completely and disable `doubleTapDragZoom` in `latlng_to_screen_point.dart`? Or do you want to have rotation disabled in more examples in general?

Reply:
Removing the `interactionOptions` property here and disabling `doubleTapDragZoom` in the other file seems fine to me 👍

Reply:
> Removing the interactionOptions property here and disabling doubleTapDragZoom in the other file seems fine to me 👍

I wasn't aware that the `markers.dart` screen makes use of the onTap callback, sorry about that. If the user clicks on the map a marker on that position gets added. Therefore I kept double tap gestures disabled on that page. Let me know if you still prefer to have your requested changes implemented or if you are fine with how it is now.

Reply:
Resolving this conversation for now, please feel free to reopen in case changes are required.